data/mods/No_Lift_Requirements/modinfo.json

```diff
@@ -3,6 +3,7 @@
     "type": "MOD_INFO",
     "ident": "disable_lifting",
     "name": "No Lifting Reqs",
+    "authors": "Wad67",
     "description": "Strength checks and lifting qualities are no longer required for installing or removing vehicle parts.",
```

Comment:
I don't know why but copy+pasting code from GitHub's UI includes -+ signs like this and line numbers.